doc/guide.md

```diff
@@ -2750,6 +2750,13 @@ there happens to be one, Okapi will remove it.  Note that it may end
 with a path like in `https://folio.example.com/okapi`.
 * `dockerUrl`: Tells the Okapi deployment where the Docker Daemon
 is. Defaults to `unix:///var/run/docker.sock`.
+* `dockerRegistries`: List of registries to use for Docker image pull. The
+value is a JSON array of of objects where each object may have the following properties
+`username`, `password`, `email`, `serveraddress` and `registry`. The first 4 properties
+are passed as autentication to the Docker registry - refer to
+[Docker Authenticaton](https://docs.docker.com/engine/api/v1.40/#section/Authentication).
```

Comment:
Can you add an example that also shows how we support unauthenticated pulls?

Reply:
That would be an empty object as one of the element for dockerRegistries .. is that the example you want? (And also the existing behavior of Okapi, when no dockerRegistries is defined).

Reply:
Mentioned in doc now